pom.xml

```diff
@@ -387,6 +435,29 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.apache.felix</groupId>
```

Comment:
Should we already merge this OSGi specific changes? What do you think?

Reply:
Same as above

>🤔 wasn't sur about it since it keeps track of only versions used in the osgi integration, but i after rethinking it it might be better to keep it separated in the osgi branch.